frontend/package.json

```diff
@@ -26,17 +26,17 @@
     "verify": "yarn build && yarn lint && yarn test"
   },
   "dependencies": {
-    "@openshift/dynamic-plugin-sdk": "^2.0.1",
+    "@openshift/dynamic-plugin-sdk": "^3.0.0",
```

Comment:
We're also on `"@openshift/dynamic-plugin-sdk": "^4.0.0"` now.

Reply:
The chrome and scalprum packages do not use the latest SDK yet. That's why using v3.

Reply:
Updated to latest.